src/jit/emitxarch.cpp

```diff
@@ -4440,12 +4440,16 @@ void emitter::emitIns_R_R_S_I(
 //
 // Returns:
 //    opReg encoded in imm[7:4]
-static int encodeXmmRegAsIval(regNumber opReg)
+static int8_t encodeXmmRegAsIval(regNumber opReg)
```

Comment:
I'm not sure it's necessary to change the signature, especially since the return value is used as an int.

Reply:
Fair enough. Will update.

Reply:
Fixed.